features/twitch-integration/index.md

```diff
@@ -0,0 +1,124 @@
+3. [Create a Reward](https://dev.twitch.tv/docs/drops/#create-a-reward)
+   1. If you have not yet created an item in Catena that you want to assign as the Twitch Drop, you should do so now.
+      - _Dev Q: Do we have a page for this?_
+      - TBD: Dashboard doc for this?
+   2. Once that item has been created, mark down its Catena Catalog Item Id.
+   3. Back in the Twitch reward creation, there is an optional "Reward ID" field. Fill that field in with the Catena Catalog Item Id.
+4. [Create a Drops Campaign](https://dev.twitch.tv/docs/drops/#create-a-drops-campaign)
+   1. Be sure to check the `Test the Drop` section of the documentation. You'll need at least two accounts:
+      1. One of which will be the Creator that will stream the game.
+      2. The other of which will be the viewer that will receive the Twitch Drop(s).
+
+## Catena Setup
+1. In the `catena-tools-core` project, open `appsettings.json` (or any of its variations).
+2. In the `Entitlements` section, if it does not exist, add an `ExternalItemProviders` field. An empty block will look like this:
+```json
+"ExternalItemProviders": {
+  "TwitchDropsProvider": {
+    "Database": {
+      "ConnectionString": "Data Source=Build/SqliteDatabase/twitch.db;Foreign Keys=True"
+    },
+    "ClientID": "",
+    "Secret": "",
+    "GameID": "",
+    "OrganizationID":  "",
+    "WebHookSecret": "",
+    "SkipSignatureVerification": false
+  }
+}
+```
+3. Fill in the `Client ID`, `Secret`, `GameID`, and `OrganizationID` fields with the information recorded from the [Twitch Setup](#Twitch-Setup) section.
+4. Fill in the `WebHookSecret` with a 10-100 character string of your choosing.
+5. Leave the `SkipSignatureVerification` field as `false`.
+   - Flipping this field to `true` will activate an emulated Twitch Drop layer meant for testing, and will not connect to Twitch proper.
+6. Ensure that the Twitch viewer account that will receive Twitch Drops is connected to a Catena Account.
+   - For instructions on how to do the above, please see [Catena Account Linking](this-page-doesn't-exist)
+     - _Dev Q: Does this page exist?_
+
+### Testing and Verifying the Catena Pipeline
+If you want to ensure that Catena is working as expected, in Step 5 of [Catena Setup](#catena-setup), you can flip the `SkipSignatureVerification` flag to `true`.
+Additionally, you will need to include the following field in the `PreferredImplementations` section:
+```json
+"PreferredImplementations": {
+   ...
+   "ITwitchDropsWrapper": "EmulTwitchDropsWrapper"
+   ...
+}
+```
+After doing the above, and assuming that you have already created a Catena Item as per Step 3.1 of [Twitch Setup](#twitch-setup), you'll need to trigger the fake Twitch Drop from an outside source.
+For now, we'll use Postman.
+   - Dev Note: Should we offer some kind of http testing section in the dashboard or is that overkill?
```

Comment:
Yes this might be something we can do in the future w/ some recognition of the `EmulTwitchDropsWrapper`.